lua/telescope/actions/init.lua

```diff
@@ -420,6 +420,33 @@ actions.smart_send_to_qflist = function(prompt_bufnr)
   end
 end
 
+actions.complete = function(prompt_bufnr)
+  local tags = action_state.get_current_picker(prompt_bufnr).sorter.tags
+  local delimiter = action_state.get_current_picker(prompt_bufnr).sorter._delimiter
+
+  if not tags then return end
+
+  local symbols = {}
+  -- :sub(2, -1): remove preceding ">"
+  local line = vim.api.nvim_buf_get_lines(prompt_bufnr, 0, 1, false)[1]:sub(2,-1) or ""
```

Comment:
Cool feature. I will look at it closer later.
But thats one thing i immediately noticed. This will not work if the prompt is different or not existing.
There is `action_state.get_current_line()` or so that will do give you the line without `prompt_prefix`

Reply:
Thanks! Changed accordingly :)